packages/autorest.typescript/test/rlcIntegration/generated/dpgCustomization/package.json

```diff
@@ -21,7 +21,7 @@
     "LICENSE",
     "review/*"
   ],
-  "sdk-type": "client",
+  "sdk-type": "mgmt",
```

Comment:
This is expected, autorest will set openapi-type as arm. by default https://github.com/Azure/autorest/blob/fa08f8cc5047c03320be8928d7f62d9235238f0a/packages/libs/configuration/resources/default-configuration.md?plain=1#L10 
and we use both azureArm and openapi-type === arm as isArm condition.

Another reason is for sdk-type related information, it will only get generated when azureSdkForJs is true.